src/app/base/components/ModelActionForm/ModelActionForm.tsx

```diff
@@ -7,14 +7,18 @@ import type { EmptyObject } from "@/app/base/types";
 type Props = {
   modelType: string;
   message?: string;
+  showSubtext?: boolean;
+  subtext?: string;
```

Comment:
Why not simply output everything as a single string for simplicity? I think we can safely drop the light text styling for the sake of reducing complexity.

As a side note, next time in a similar scenario I'd recommend to simplify, `showSubtext` prop is redundant in this case. As `subtext` is already optional we can assume showSubtext will always be false if subtext is falsy (undefined) and remove it.